cypress/support/organizations.js

```diff
@@ -0,0 +1,16 @@
+import { Button, Select, TextField } from '../../interactors';
+
+Cypress.Commands.add('createOrganization', ({ name, code, status }) => {
+  const saveButton = Button({ id: 'organization-form-save' });
+
+  cy.expect(saveButton.has({ disabled: true }));
```

Comment:
not sure we need expectation that save button is enabled/disabled in createOrg command